docs/production/scim.md

```diff
@@ -1,34 +1,65 @@
-# SCIM
+# SCIM provisioning
 
-Zulip has preliminary support for user provisioning and deprovisioning
-via the SCIM protocol, with Okta being the only currently supported provider.
-Support for other providers is expected to be added in the future.
+Zulip has beta support for user provisioning and deprovisioning via
+the SCIM protocol. In SCIM, a third-party SCIM Identity Provider (IdP)
+acts the SCIM client, connecting to the service provider (your Zulip
+server).
 
-Groups are currently not supported.
+See the [SCIM help center page](https://zulip.com/scim) for
+documentation on SCIM in [Zulip Cloud](https://zulip.com) as well as
+detailed documentation for how to configure some SCIM IdP providers.
+
+Synchronizing groups via SCIM is currently not supported.
 
 ## Server configuration
 
-The server-side configuration is straightforward:
+The Zulip server-side configuration is straightforward:
+
+1. Pick a client name for your SCIM client. This name is internal to
+   your Zulip configuration, so the name of your IdP provider is a
+   good choice. We'll use `okta` in the examples below.
+1. First a SCIM client entry needs to be added to the database. Run
+   `manage.py add_scim_client <client name> -r <subdomain>`. For
+   example, if your organization is hosted at on a subdomain
+   (`subdomain.zulip.example.com`):
```

Comment:
I added the examples here to avoid folks putting a lot of angle brackets into their configuration by accident.